FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -188,6 +188,14 @@ public static function cleanStringProvider()
         ];
     }
 
+    public function categoryNamesProvider()
+    {
+        return [
+            'Root category name without children' => [1, ' Root '],
+            'Category name with parent' => [5, ' Genusswelten '],
```

Comment:
For this case, also wrap the parent category with spaces to make sure that all of them were trimmed

Reply:
@ihanli wrap the parent category as in? withe the `|` symbol?